app/presenters/outcome_presenter.rb

```diff
@@ -31,40 +55,60 @@ def has_calendar?
     calendar.present?
   end
 
-  def has_body?
-    use_template? || super()
-  end
-
   def body
-    if use_template?
-      view_context = @state.dup
```

Comment:
sounds like [Object#freeze](http://ruby-doc.org/core-2.2.2/Object.html#method-i-freeze) except that it will raise an error if the state is being modified.